SofascoreAcademy/SofascoreAcademy/LeagueInfoView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  LeagueNameView.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 12.03.2024..
+//
+
+import Foundation
+import SnapKit
+import UIKit
+import SofaAcademic
+
+class LeagueInfoView: BaseView {
+
+    private var countryName: String = ""
+    private var leagueName: String = ""
+    private var leagueLogo: String?
+    private var arrow: String = "pointer"
+
+    private let stackView = UIStackView()
+    private let countryNameLabel = UILabel()
+    private let leagueNameLabel = UILabel()
+    
+    private let leagueLogoImageView = UIImageView()
+    private let arrowImageView = UIImageView()
+    
+    func update(countryName: String, leagueName: String, leagueLogo: String) {
+        self.countryName = countryName
+        self.leagueName = leagueName
+        self.leagueLogo = leagueLogo
+        
+        addViews()
+        styleViews()
+        setupConstraints()
+    }
+    
+    override func addViews() {
+        addSubview(stackView)
+        addSubview(leagueLogoImageView)
+        stackView.addArrangedSubview(countryNameLabel)
+        stackView.addArrangedSubview(arrowImageView)
+        stackView.addArrangedSubview(leagueNameLabel)
+    }
+
+    override func styleViews() {
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        countryNameLabel.text = countryName
+        countryNameLabel.font = fonts.RobotoBold
+        
+        leagueNameLabel.text = leagueName
+        leagueNameLabel.textColor = colors.surfaceLv2
+        leagueNameLabel.font = fonts.RobotoBold
+
+        if let leagueLogo = leagueLogo {
```

Comment:
Pretpostavljam da je ovo rjesenje za pitanje sa slacka? Snasao si se i super, ali treba li ti taj dio koda tu uopce?
Ti si inicijalizirao UIImageView() i u styleViews obzirom da nemas podatke jos bas te briga za to koji je njegov UIImage.